frontend/src/app/components/ErrorPage.js

```diff
@@ -7,7 +7,7 @@ export default class ErrorPage extends React.Component {
   render() {
     return (
       <div className="full-page-wrapper space-between">
-        <Header />
+        <Header openWindow={this.props.openWindow} sendToGA={this.props.sendToGA} />
```

Comment:
Updated commit to make this change - kind of obscures the particular things passed in, but it's nice consistent boilerplate